ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureActivity.java

```diff
@@ -13,7 +13,7 @@
 
 public class ThreeDSecureActivity extends AppCompatActivity implements CardinalValidateReceiver {
```

Comment:
What does this class do? Could we add a doc comment? I'm confused about how this is used in relation to `ThreeDSecureClient`.

Reply:
I guess we could make a doc string? It's the intermediary between Cardinal and the SDK when using 3DS v2 flow.

Reply:
Why is it public?

Reply:
That's a good call. We looked into this and it turns out to be a Java / Android limitation. `ThreeDSecureActivity` needs to be publicly accessible by the Android system in order to start the activity, which in turn starts Cardinal. We do believe this can be removed in the future, once Cardinal provides us with an update where the response is returned via an Android Activity result.

Reply:
> Could we add a doc comment? 

We're going to go ahead and add the doc strings! 🙌

Reply:
Awesome, thanks for the clarification!